multiplicationTable/ViewController.swift

```diff
@@ -16,21 +16,19 @@ class ViewController: UIViewController, UITableViewDelegate, UITableViewDataSour
     var num = 0
     
     var numbers = ["1", "2", "3", "4", "5", "6", "7", "8", "9", "10"]
```

Comment:
You can use indexPath.row directly with +1, this will avoid creating one extra array

Reply:
Sir, I am getting a fatal error when I add +1 and it is saying that:Index out of range.